### ISSUE TITLE
best model weight

### ISSUE THREAD

**Issue by hescluke** [Hydragon516/DATA#5]
I got about 0.77 J score after pretrain stage, but when I began fintuning, the validation result showed that this best pretrained model weight got 0 J score on DAVIS validation set. It seems that the model began learning from 0 score not 0.77 score on finetune stage. Could you please tell me how to resolve this problem? Or could you please upload the best model weight on DAVIS directly? Thanks!
